### PR TITLE
added modal to ask for matomo

### PR DESCRIPTION
**Pull request by LianaHus**
the commented out code should be opened after testing

### CODE REVIEW

File: apps/remix-ide/src/app.js [hunk 1]
```diff
@@ -347,6 +349,34 @@ Please make a backup of your contracts and start using http://remix.ethereum.org
...
+        label: "Sure",
+        fn: () => {
+          settings.updateMatomoAnalyticsChoice(true)
+          _paq.push(['forgetUserOptOut'])
+        }
+      },
+      {
+        label: 'Decline',
+        fn: () => {
+          settings.updateMatomoAnalyticsChoice(false)
```
Comment (yann300) [inline apps/remix-ide/src/app.js:373]: that is not how should be done an API call, and surely that shouldn't be put in the remix plugin API

**Review by ryestew — COMMENTED** [review]
Here is a rewrite of this text:

```
 <p>Remix IDE uses <a href="https://matomo.org">Matomo</a>, an opt-in, open source data analytics platform to improve the UX & UI of our website.</p>
       <p>We realize that our users have sensitive information in their code and that the privacy of our users must be protected, therefore, we do not collect nor store any personally identifiable information (PII).</p>
        <p>All data collected through Matomo is stored at our own server - no data is given to third parties.</p>
<p>For more information about Matomo on Remix IDE please see our blog post.</p>
<p>You can change your choice in the Settings panel anytime.</p>
<p>Will you join us in this initiative to improve Remix?</p>
```
The html above includes the line about the medium post - which is not yet published - so you could leave that out for now.  (But it will be published before the release.)

**Comment by ryestew** [comment]
When I load this branch, the screen flashes from black to white for an instant.  Whether the modal loads or not.  I don't see this happening in master.

**Review by ryestew — APPROVED** [review]

**Comment by LianaHus** [comment]
> Here is a rewrite of this text:
> 
> ```
>  <p>Remix IDE uses <a href="https://matomo.org">Matomo</a>, an opt-in, open source data analytics platform to improve the UX & UI of our website.</p>
>        <p>We realize that our users have sensitive information in their code and that the privacy of our users must be protected, therefore, we do not collect nor store any personally identifiable information (PII).</p>
>         <p>All data collected through Matomo is stored at our own server - no data is given to third parties.</p>
> <p>For more information about Matomo on Remix IDE please see our blog post.</p>
> <p>You can change your choice in the Settings panel anytime.</p>
> <p>Will you join us in this initiative to improve Remix?</p>
> ```
> 
> The html above includes the line about the medium post - which is not yet published - so you could leave that out for now. (But it will be published before the release.)

yes I'll add it once it is published. I need a link

**Pull request merged**